arrow-libs/optics/arrow-optics-ksp-plugin/src/main/kotlin/arrow/optics/plugin/internals/processor.kt

```diff
@@ -135,10 +135,10 @@ internal fun KSClassDeclaration.getConstructorTypesNames(): List<String> =
 
 internal fun KSType.qualifiedString(): String = when (declaration) {
   is KSTypeParameter -> {
-    val n = declaration.simpleName.asString()
+    val n = declaration.simpleName.asString().sanitizeDelimited()
     if (isMarkedNullable) "$n?" else n
   }
-  else -> when (val qname = declaration.qualifiedName?.asString()) {
+  else -> when (val qname = declaration.qualifiedName?.asString()?.sanitizeDelimited()) {
```

Comment:
It might read better if `asStringSanitized` was an extension of `Name` (but if you prefer this way, it's also fine)

Reply:
I agree, it would be more readable as well as limit the scope of the function to relevant parts i.e KSName rather than any String. Will change the extension signature and apply it where applicable.